shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/repository/SurveyRepository.kt

```diff
@@ -10,4 +10,6 @@ interface SurveyRepository {
         pageSize: Int,
         isForceLatestData: Boolean = false
     ): Flow<List<Survey>>
+
+    fun getSurvey(id: String): Flow<Survey>
```

Comment:
I think this request is for getting all survey questions, so what do you think if we name it `getSurveyDetail`?

Reply:
No concerns from my side, I will update this

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/110/commits/3dd86b6dd066888cac19cd03ccc631346926311b